lib/Sema/CSSimplify.cpp

```diff
@@ -15650,6 +15649,12 @@ bool ConstraintSystem::recordFix(ConstraintFix *fix, unsigned impact,
   if (auto impactScoreKind = fix->impact())
     increaseScore(*impactScoreKind, fix->getLocator(), impact);
 
+  if (isDebugMode()) {
+    auto &log = llvm::errs();
+    log.indent(solverState ? solverState->getCurrentIndent() : 0)
+      << "(fix has score " << impact << ")\n";
```

Comment:
I think this is duplicate information because `increaseScore` already prints this with more information.